### PR TITLE
refactor(v)!: update repository

### PR DESCRIPTION
**Pull request by amaanq**
Old one had a memory leak and was archived

relevant convo https://github.com/vlang/vls/pull/446

### CODE REVIEW

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -1600,12 +1600,12 @@ list.uxntal = {
...
 list.v = {
   install_info = {
-    url = "https://github.com/vlang/vls",
+    url = "https://github.com/v-analyzer/v-analyzer",
     files = { "src/parser.c", "src/scanner.c" },
     location = "tree_sitter_v",
   },
-  filetype = "vlang",
-  maintainers = { "@kkharji" },
+  filetype = "v",
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:1607]: That's an unrelated change. Any reason for it?

**Review by clason — COMMENTED** [review]
Also needs a lockfile update.

**Comment by lucario387** [comment]
Bit ootl here, but shouldn't the URL be https://github.com/v-analyzer/v-tree-sitter ? Or is there a reason why there are two separate tree-sitter parsers for v

**Comment by clason** [comment]
foo-tree-sitter != tree-sitter-foo

(the former is bindings for tree-sitter in the foo language, the latter a tree-sitter parser for the foo language)

**Comment by lucario387** [comment]
> (the former is bindings for tree-sitter in the foo language, the latter a tree-sitter parser for the foo language)

Yep, I didn't read that correctly at first. Should have been more aware

**Review by lucario387 — APPROVED** [review]
